### PR TITLE
BLD: Add multiprocessing option through emcee and make it default.

### PR DESCRIPTION
**Pull request by olivia-higgins**
BLD: Add user input for cores.

### CODE REVIEW

**Comment by olivia-higgins** [comment]
@bocklund

File: espei/run_espei.py [hunk 1]
```diff
@@ -135,25 +135,31 @@ def main():
...
             client = MPIPool()
             if not client.is_master():
-                logging.debug(
+                logging.warning(
                     'MPIPool is not master. Waiting for instructions...')
                 client.wait()
                 sys.exit(0)
             logging.info("Using MPIPool on {} MPI ranks".format(client.size))
-        elif mcmc_settings['scheduler'] == 'dask':
+        if mcmc_settings['scheduler'] == 'dask':
```
Comment (bocklund) [inline espei/run_espei.py]: Any reason for changing this from `elif` to `if`?

File: espei/run_espei.py [hunk 2]
```diff
@@ -135,25 +135,31 @@ def main():
             # code recommended by emcee: if not master, wait for instructions then exit
             client = MPIPool()
             if not client.is_master():
-                logging.debug(
+                logging.warning(
```
Comment (bocklund) [inline espei/run_espei.py]: Make sure your commits are rebased on top of the HEAD of PhasesResearchLab/ESPEI so that my previous commits are not undone.

**Comment by bocklund** [comment]
Also in the `writing_input.rst` doc, document that 

1. emcee is the new default scheduler
2. The cores option. That it can be can be used for emcee and dask. That it doesn't take effect for MPIPool.
3. Consider the consequences and handling of passing more cores than the user has

File: espei/run_espei.py [hunk 3]
```diff
@@ -131,16 +131,19 @@ def main():
         save_interval = mcmc_settings.get('mcmc_save_interval')
         # scheduler setup
         if mcmc_settings['scheduler'] == 'MPIPool':
+            # check that cores is not an input setting
+            if mcmc_settings['cores'] != None:
+                logging.info("MPI does not take the cores input setting.")
```
Comment (bocklund) [inline espei/run_espei.py]: I think warning is more fitting. See http://espei.org/en/latest/contributing.html#logging

**Comment by bocklund** [comment]
@olivia-higgins any updates here?

**Comment by bocklund** [comment]
Closed and reopened as #38

**Pull request closed without merging**